website/common/src/gear/gear-events.ts

```diff
@@ -0,0 +1,12 @@
+export enum GEAR_EVENT {
+  MESSAGE_ENQUEUED = 'MessageEnqueued',
+  USER_MESSAGE_SENT = 'UserMessageSent',
+  USER_MESSAGE_READ = 'UserMessageRead',
+  MESSAGES_DISPATCHED = 'MessagesDispatched',
+  MESSAGE_WAITED = 'MessageWaited',
+  MESSAGE_WAKEN = 'MessageWaken',
+  CODE_CHANGED = 'CodeChanged',
+  PROGRAM_CHANGED = 'ProgramChanged',
+  DEBUG_DATA_SNAPSHOT = 'DebugDataSnapshot',
+  DEBUG_MODE = 'DebugMode',
```

Comment:
There is no need to keep all of these events here. You need only `MessageEnqueued`, `UserMessageSent`, `MessagesDispatched`, `CodeChanged`, `ProgramChanged` and `DatabaseWiped`